docs/tasks/administer-cluster/pvc-protection.md

```diff
@@ -0,0 +1,36 @@
+---
+approvers:
+- msau42
+- jsafrane
+title: Persistent Volume Claim Protection
```

Comment:
This page is a task. Taks has steps with concrete examples. There are no steps below, just installation prerequisites. I'd expect some PVC and Pod creation, PVC deletion (+ show how PVC is not deleted) and Pod deletion (+ show that PVC and PV is deleted).

Or don't create a task page and put the info somewhere else.

Reply:
@jsafrane It looks like creation (line 31) and deletion (line 81) are both covered, so a task is the most appropriate template here.